### PR TITLE
Niad 192 publish tech content

### PR DESCRIPTION
**Pull request by MattLewis80**
Added img folder, NIA HLD Arch image and referenced in README

### CODE REVIEW

File: NHAIS/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Onboarding
+As outlined in section 2, the purpose of each adaptor is to enable access to national NHS Systems.  As an enabler, the adaptor will need to be configured to integrate with the relevant NHS environment(s).  Therefore you will need to register your intention to use adaptors through the standard NHS onboarding process which can be found here within the NHS Operations overview. 
+
+
+## Installation & Configuration inc dependencies 
+
+### Open Test and Path To Live Access
+
+In order to perform end-to-end testing of the MHS Adaptor, you will require access to an NHS Digital test environment. The MHS adaptor connects to the following two test endpoints which are available in an NHS Digital test environment:
```
Comment (michael-kainos) [inline NHAIS/README.md:14]: We mention MHS in this NHAIS document, is that ok?

File: NHAIS/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+NHAIS is a system that allows General Practice (GP) Surgeries to keep their patient registration and demographics data in sync with the regional Health Authorities (HA). Since the creation of this service the regional or area health authorities (approx 80) have since been replaced by a fewer number of successor organisations. There is however still a notion of every GP Practice’s patient being registered with one of the HAs.
+
+
+## Onboarding
+As outlined in section 2, the purpose of each adaptor is to enable access to national NHS Systems.  As an enabler, the adaptor will need to be configured to integrate with the relevant NHS environment(s).  Therefore you will need to register your intention to use adaptors through the standard NHS onboarding process which can be found here within the NHS Operations overview. 
+
+
+## Installation & Configuration inc dependencies 
+
+### Open Test and Path To Live Access
```
Comment (michael-kainos) [inline NHAIS/README.md:12]: Should the OpenTest and PTL access sit in the overarching document as this is a one time thing not adapter specific?

File: NHAIS/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+NHS Digital provides OpenTest as an environment which can be used to perform test activities where an Health & Social Care Network (HSCN) connection is not available. 
+
+
+To set up and configure OpenTest up please use Set up NHS Digital OpenTest connection.  ***Dev work/spikes/POC’s***
+
+To set up and configure Path to Live connectivity, please see Path to Live environments.  ***Int env testing***
+
+
+
+## MHS Services and how to Deploy them
```
Comment (michael-kainos) [inline NHAIS/README.md:26]: MHS mentioned here, is this ok?

**Review by michael-kainos — COMMENTED** [review]
@MattLewis80 please see comments

File: NHAIS/README.md (hunk 2 above)
Comment (MattLewis80) [reply NHAIS/README.md:12]: Yep!

File: NHAIS/README.md (hunk 1 above)
Comment (MattLewis80) [reply NHAIS/README.md:14]: copy and paste special

File: README.md [hunk 4]
```diff
@@ -1,34 +1,36 @@
...
+
+[NIA Overview](img/High Level Architecture.png)
+
+ 
+A supplier is therefore required only to implement a simplified set of standard clients to the adaptor layer, integrating with a simplified and standardised set of interfaces exposed by the Adaptor layer. This set of clients is shown in the "Interface Layer" as highlighted in the diagram above.
+
+
+##Adaptors
+
+###Message Handling System (MHS)
```
Comment (michael-kainos) [inline README.md:26]: i think we can remove this also as NHAIS won't be using MHS?

**Review by michael-kainos — APPROVED** [review]

File: NHAIS/README.md (hunk 3 above)
Comment (MattLewis80) [reply NHAIS/README.md:26]: Nope, I will resolve tomorrow along with the hyperlinks

File: README.md (hunk 4 above)
Comment (MattLewis80) [reply README.md:26]: Yep, will sort tomorrow!

**Pull request merged**